test/test2.js

```diff
@@ -0,0 +1,19 @@
+import testFunction from './test-function.js';
+const test = QUnit.test;
+
+
+test('too high or too low?', function(assert) {
```

Comment:
You need to have more tests than this - you want to test for every case with the function, so should have one test for matching the number, one for too high, and one for two low. The name of the test should reflect which case you're testing for.